dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -891,8 +891,6 @@ LearnerReadSnapshot DAGStorageInterpreter::doBatchCopLearnerRead()
             if (context.getDAGContext()->is_disaggregated_task)
                 throw;
 
-            if (tmt.checkShuttingDown())
-                throw TiFlashException("TiFlash server is terminating", Errors::Coprocessor::Internal);
```

Comment:
So why this is also removed?

Reply:
I forgot to change it back. I originally deleted it to retry on the region error when the status is `stopping`. Now it is not needed, the code has been restored.